### ISSUE TITLE
Importing CSS file with CSS variables errors

### ISSUE THREAD

**Issue by mattfelten** [sass/dart-sass#1165]
```
// variables.css
:root {
	--default-border-radius: 3px;
}
```
```
// index.scss
@use 'variables.css';
```

```
TypeError: Instance of 'SassNumber': type 'SassNumber' is not a subtype of type 'SassString'


node_modules/sass/sass.dart.js 1194:17   Object.wrapException
node_modules/sass/sass.dart.js 2495:15   Object._failedAsCheck
node_modules/sass/sass.dart.js 2492:9    Rti._generalNullableAsCheckImplementation
node_modules/sass/sass.dart.js 58043:41  _SerializeVisitor0._writeReindentedValue$1
node_modules/sass/sass.dart.js 58788:12  _SerializeVisitor_visitCssDeclaration_closure.call$0
node_modules/sass/sass.dart.js 46564:23  SourceMapBuffer0.forSpan$1$2
node_modules/sass/sass.dart.js 46571:19  SourceMapBuffer0.forSpan$2
node_modules/sass/sass.dart.js 58003:12  _SerializeVisitor0.visitCssDeclaration$1
node_modules/sass/sass.dart.js 32279:22  ModifiableCssDeclaration.accept$1$1
node_modules/sass/sass.dart.js 32282:19  ModifiableCssDeclaration.accept$1
```

I'm trying to merge a few CSS files into one big one and coming up short. One of them has CSS variable declarations in it and seems to be causing the compiler to be unhappy.

Using `sass@1.26.12`

**Comment by jathak** [comment]
Actually, it looks like this was fixed in 1.27.0. Try updating to the latest version of Sass.

**Comment by mattfelten** [comment]
Oh nice! I looked through the release history but didn't see anything that stood out as related. Thanks for the info!
